README.md

```diff
@@ -61,13 +34,13 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+
+# 📖 [eHllo microverse!] <a name="Hello microverse!"></a>
```

Comment:
- [ ] Again here you were advised to correct the word from eHllo to Hello by the previous reviewer, kindly do so